tutorials/docker-compose-on-container-optimized-os.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Running Docker Compose with Docker
+description: Learn how to run the Docker Compose command using Docker on a Container-Optimized OS Google Compute Engine instance.
+author: tswast
+tags: Docker,Docker Compose,Container-Optimized OS,Compute Engine
+date_published: 2017-05-03
+---
+
+This tutorial guides you through running the Docker Compose command via Docker.
+
+## Objectives
+
+- Run Docker Compose with Docker on Container-Optimized OS.
+- Create an alias for the containerized `docker-compose` command.
+
+## Before you begin
+
+1.  Create or select a Cloud Platform project from the [Google Cloud Platform
+    console's projects page](https://console.cloud.google.com/project).
+1.  [Enable
+    billing](https://support.google.com/cloud/answer/6293499#enable-billing)
+    for your project.
+
+## Costs
+
+This tutorial uses billable components of Cloud Platform including
+
+- [Google Compute Engine](/compute/pricing)
+
+Use the [Pricing Calculator](/products/calculator/) to estimate the costs for
+your usage.
+
+## Setting up the virtual machine
```

Comment:
Maybe... What I'd really want is for Container-Optimized OS to have these steps in a quickstart.